cypress/e2e/v1.0/journeys/drug-use/drug-use.noninjectable.morethan6months.cy.ts

```diff
@@ -0,0 +1,126 @@
+describe('Origin: /drug-use', () => {
+  const destinations = {
+    landingPage: '/drug-use',
+    addDrugs: '/add-drugs',
+    drugDetails: '/drug-details',
+    drugDetailsMoreThanSix: '/drug-details-more-than-six-months',
+    drugUseHistory: '/drug-use-history',
+    drugUseHistoryAllMoreThanSix: '/drug-use-history-more-than-six-months',
+    summary: '/drug-use-summary',
+    analysis: '/drug-use-analysis',
+  }
+
+  const sectionName = 'Drug use'
+
+  before(() => {
+    cy.createAssessment()
+  })
+
+  beforeEach(() => {
+    cy.enterAssessment()
+  })
+
+  describe(`Destination: ${destinations.summary}`, () => {
+    it(`No drug use routes to "${destinations.summary}"`, () => {
+      cy.visitStep(destinations.landingPage)
+      cy.getQuestion('Has Sam ever misused drugs?').getRadio('No').clickLabel()
+      cy.assertResumeUrlIs(sectionName, destinations.landingPage)
+      cy.saveAndContinue()
+      cy.assertStepUrlIs(destinations.summary)
+      cy.assertResumeUrlIs(sectionName, destinations.summary)
+    })
+
+    // TODO when this part is done
+    // testPractitionerAnalysis(sectionName, destinations.summary, destinations.analysis)
+  })
+
+  describe(`Destination: ${destinations.landingPage}`, () => {
+    it(`"Drug use no routes to "${destinations.summary}"`, () => {
+      cy.visitStep(destinations.landingPage)
+      cy.getQuestion('Has Sam ever misused drugs?').getRadio('No').clickLabel()
+      cy.assertResumeUrlIs(sectionName, destinations.landingPage)
+      cy.saveAndContinue()
+      cy.assertStepUrlIs(destinations.summary)
+      cy.get('.govuk-back-link').should('not.exist')
+      cy.assertResumeUrlIs(sectionName, destinations.summary)
+    })
+
+    it(`"Drug use yes routes to "${destinations.addDrugs}"`, () => {
```

Comment:
This belongs to the `describe` below, because it's a new destination

Reply:
Fixed in [321c26d](https://github.com/ministryofjustice/hmpps-strengths-based-needs-assessments-ui/pull/828/commits/321c26d82730e98e5cc1888c7b0e63b92f7b5cb1)